_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -3,59 +3,61 @@ layout: post
 title: "Team Topologies Inspired Whitelist App Development"
 ref: tt-whitelist
 lang: en
-author: "IT Strategy Team"
+author: "Jayson McIntosh"
 date: "2020-05-20"
 last_modified: "2020-05-20"
 excerpt_separator: <!--more-->
 ---
 
+A summary of the ideas and concepts used by the IT Strategy team and DTS to deliver the Whitelist application using DevOps inspired principles and ideas. By using this model, the team was able to demo an application to the CIO within 10 days of starting work, and had users on the cloud testing the application within 2 hours of PB being available.
```

Comment:
What is DTS?

Reply:
Also I'm assuming that PB is protected B but probably for clarity it would be good to spell this out.

Reply:
I have no idea what DTS stands for ... it's a team. Good point! I shall ask the team and add this.

Reply:
Fixed!